src/api/operations/staffAssignment.fragments.graphql

```diff
@@ -40,11 +40,9 @@ fragment StaffAssignmentWithClients on StaffAssignment {
   assignedAt
   unassignedAt
   household {
-    id
+    ...HouseholdWithEnrollmentFields
```

Comment:
This looks like it resolves ssn and dob (and access object) for all household members which we don't need for staff assignment. I think we should avoid that since it's not necessary in this case and it is highly sensitive.

Maybe just inline the fields needed for staff assignment?